tests/lib/devicetree/api/prj.conf

```diff
@@ -1 +1,2 @@
 CONFIG_ZTEST=y
+CONFIG_ZTEST_NEW_API=y
```

Comment:
?

Reply:
Sorry, other test changes got into the commit by mistake.  Working on cleaning this.